src/main/java/elements/PageElementImpl.java

```diff
@@ -0,0 +1,131 @@
+package elements;
+
+import factory.DriverFactory;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class PageElementImpl implements PageElement {
+    protected final int DEFAULT_VISIBILITY_TIME = 20;
+    protected WebElement element;
```

Comment:
replace this element with method getWebElement()

Reply:
you meen create getter for WebElement?